node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+const fileToCopy = process.argv[2];
+const newFileName = process.argv[3];
+
+async function copy() {
+  try {
+    const originalContents = await readFile(fileToCopy, 'utf8');
```

Comment:
It would be best to do the readFile without an encoding. With it, you can only copy text files. If you leave it off you'll be able to copy images, etc., as well.